recipes/sqlitecpp/all/conandata.yml

```diff
@@ -15,3 +15,6 @@ patches:
   "2.5.0":
     - base_path: "source_subfolder"
       patch_file: "patches/2.5.0/0001-conan.patch"
+  "3.1.1":
+    - base_path: "source_subfolder"
+      patch_file: "patches/3.1.1/0001-conan.patch"
```

Comment:
Can this have a more descriptive name? so the next contributors knows

```suggestion
      patch_file: "patches/3.1.1/0001-cmake-add-omit-load-extensions.patch"
```

Reply:
sure, but im following other packages

this further delays packages from entering